### PR TITLE
Feat : avatar and badges done

### PR DESCRIPTION
**Pull request by anuj-kumary**

### CODE REVIEW

**Review by lomesshh — APPROVED** [review]
Looks good !

**Comment by Hnh013** [comment]
looks good to merge , great work !

**Review by Hnh013 — APPROVED** [review]
approving changes now!

**Comment by Rohanmond** [comment]
Look good, great!

**Review by Rohanmond — APPROVED** [review]

File: components/Badge/badge.css [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    display: flex;
+    align-self: flex-end;
+    text-align: center;
+    margin-right: 1rem;
+  }
+  .badge__avatar {
+    width: 1.5rem;
+    height: 1.5rem;
+    border-radius: 50%;
+    border: 2px solid #fff;
```
Comment (Parul1999) [inline components/Badge/badge.css:16]: Don't hard-code colours.

File: components/Badge/badge.css [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  }
+  .badge__avatar-lg {
+    height: 8rem;
+    width: 8rem;
+    border-radius: 50%;
+  }
+  .status-online {
+    background-color: #1b7052;
+    position: absolute;
+    bottom: 4%;
```
Comment (Parul1999) [inline components/Badge/badge.css]: Use same units.

File: components/Badge/badge.html [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+      referrerpolicy="no-referrer"
+    />
+  </head>
+  <body>
+    <div class="content-heading">Profile Avatar</div>
+    <div class="badge-container">
+      <div class="badge">
+        <img class="avatar__img badge__avatar-lg"
+        src="https://st4.depositphotos.com/1001992/21109/i/600/depositphotos_211096346-stock-photo-confident-man-white-shirt-posing.jpg"
+        alt=""" />
```
Comment (Parul1999) [inline components/Badge/badge.html]: alt = ""

**Review by Parul1999 — CHANGES_REQUESTED** [review]
1)Use same units iin CSS
2) don't hard-code colurcode.
3) Use semantic tags

File: components/Badge/badge.html [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.10.0/css/all.min.css"
+      integrity="sha512-PgQMlq+nqFLV4ylk1gwUOgm6CtIIXkKwaIHp/PAIWHzig/lKZSEGKEysh0TCVbHJXCLN7WetD8TFecIky75ZfQ=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+  </head>
+  <body>
+    <div class="content-heading">Profile Avatar</div>
```
Comment (Parul1999) [inline components/Badge/badge.html:18]: Use Semantic Tags

**Review by Parul1999 — APPROVED** [review]

**Pull request merged**